task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -49,8 +49,9 @@ spec:
           cpu: 10m
       script: |
         #!/usr/bin/env bash
-        set -o pipefail
+        set -euo pipefail
         source /utils.sh
+        trap 'handle_error $(results.TEST_OUTPUT.path)' EXIT
 
         declare -a ALLOWED_BASE_IMAGES=("registry.redhat.io/openshift4/ose-operator-registry")
 
```

Comment:
on line 95, 96; this won't work anymore
```
        conffolder=$(cat ../inspect-image/image_inspect.json | jq -r '.Labels ."operators.operatorframework.io.index.configs.v1"')
        if [ $? -ne 0 ]; then
```
should be replaced to

```
        status=0
        conffolder=$(jq -f ../inspect-image/image_inspect.json -r '.Labels ."operators.operatorframework.io.index.configs.v1"') || status=$?
        if [ $status -ne 0 ]; then
```
As I pointed out several times here:
https://redhat-internal.slack.com/archives/C034UFMUSV6/p1695224381051229?thread_ts=1694681340.976039&cid=C034UFMUSV6

Reply:
Fixed. Thanks.